<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>characterPad</class>
 <widget class="QWidget" name="characterPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>characterPad</string>
  </property>
  <widget class="MPushButton" name="pushButton_Q">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>q</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_W">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>w</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_E">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>e</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_R">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_T">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Y">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_U">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>u</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_I">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>i</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_O">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>o</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_P">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>p</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_A">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>a</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_S">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>s</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_D">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>d</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_F">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>f</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_G">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_H">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_J">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>j</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_K">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_L">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>50</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_M">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Z">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>z</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_X">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_C">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_V">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_B">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_N">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>75</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Space">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>271</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>空  格</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Enter">
   <property name="geometry">
    <rect>
     <x>654</x>
     <y>150</y>
     <width>141</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>确  认</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>取  消</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_numPad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>数字键盘</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Backspace">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>100</y>
     <width>111</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>删  除</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Caps">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>91</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Caps</string>
   </property>
  </widget>
  <widget class="MPushButton" name="pushButton_Shift">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>121</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">mPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
